lockfile/plugin-commands-audit/src/audit.ts

```diff
@@ -221,6 +221,7 @@ ${JSON.stringify(newOverrides, null, 2)}`,
       ['Package', advisory.module_name],
       ['Vulnerable versions', advisory.vulnerable_versions],
       ['Patched versions', advisory.patched_versions],
+      ['Paths', advisory.findings.map(({ paths }) => paths).flat().join('\n')],
```

Comment:
In workspaces, Paths output like this:

```
{
          "version": "9.6.0",
          "paths": [
            "packages____bar__>ava>update-notifier>latest-version>package-json>got"
          ]
        },
        {
          "version": "8.3.2",
          "paths": [
            "packages__qar>got"
          ]
```

Not sure how to convert `packages____bar__` in path info to the import id(`packages/__bar__`) or  package name